policyengine_us/variables/gov/states/md/tax/income/credits/eitc/non_refundable/md_married_or_has_child_non_refundable_eitc.py

```diff
@@ -0,0 +1,24 @@
+from policyengine_us.model_api import *
+
+
+class md_married_or_has_child_non_refundable_eitc(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "MD non-refundable EITC for filers who are married or have qualifying child"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://casetext.com/statute/code-of-maryland/article-tax-general/title-10-income-tax/subtitle-7-income-tax-credits/section-10-704-effective-until-6302023-for-earned-income"  # (c)(1)
+    defined_for = StateCode.MD
```

Comment:
```suggestion
    defined_for = "md_qualifies_for_unmarried_childless_eitc"
```

Reply:
This is not defined for "md_qualifies_for_unmarried_childless_eitc" but the opposite one